tools/maxquant/mqwrapper.xml

```diff
@@ -0,0 +1,265 @@
+<tool id="mqwrapper" name="maxquant" version="@VERSION@">
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <requirements>
+    <requirement type="package" version="@VERSION@">maxquant</requirement>
+  </requirements>
+  <command detect_errors="exit_code"><![CDATA[
+    python3 $__tool_directory__/mqwrapper.py
```

Comment:
all paths or all text parameters needs to be single-quoted - for security reasons.